src/libraries/System.ServiceProcess.ServiceController/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
Would it make sense to simply these compatibility suppressions files using the new feature where  u dont need to specify the target in this pr ?

I am fine it doing it in a separate pr but we should create an issue to track that work especially in case of type forwards.

Reply:
ha, we commented the same thing at the same time 😄